### ISSUE TITLE
Potential collaboration

### ISSUE THREAD

**Issue by Tokazama** [TRIImaging/BIDSTools.jl#9]
Saw this in the General registry and thought I'd see if you're interested in collaborating. I'm trying to finalize the the first PR to [NeurCore.jl](https://github.com/JuliaNeuroscience/NeuroCore.jl), which is intended to be a common interface to neuroimaging data for Julia. At this point nearly all of it is directly based off of what is in the current BIDs entities.

I was excited to see this because it's unlikely I will have time to get the file system and BIDS validator like tools up. I'm hoping to push what I'm working on in NeuroCore into a complete refactor of NIfTI.jl very soon so this could be a good way to maintain common syntax.

**Comment by c42f** [comment]
Hi, thanks for asking!

We're largely doing MR spectroscopy work at the moment but also have a lot of supporting dicom imagery which we convert to BIDS using heudiconv. So I'm not sure how much we'll end up using NeuroCore, but I'll be happy to glance over the initial design and provide some feedback :-)

**Comment by Tokazama** [comment]
Gotcha. Unfortunately spectroscopy is one of the few modalities I haven't personally worked with. I see that it has a proposal to be formally incorporated into BIDS [here](https://github.com/bids-standard/bids-specification/blob/6c17c09e38e11e7d73647b661b438dfd6d66a102/src/06-extensions.md). I'd be happy to implement an interface to the appropriate BIDS entities wherever it would be beneficial.

Edit:
Just saw that you are part of the BEP conversation.

**Comment by c42f** [comment]
Yeah, that BEP appears somewhat stalled, but I poked my nose in there and made some recommendations, for whatever they're worth :-)
